### PR TITLE
Rename section to "What are smart contracts"

### PR DESCRIPTION
**Pull request by rootulp**
## Description

This sections explains smart contracts. dapps are explained on the following page: https://ethereum.org/en/developers/docs/dapps/#definition-of-a-dapp

### CODE REVIEW

**Review by wackerow — APPROVED** [review]
@rootulp Yeah, this paragraph is a little bit of both isn't it. I'd have to agree that it seems better labeled as 'smart contracts' instead of dapps, especially with the dapp explanation on the next page. Thanks!

File: src/content/developers/docs/intro-to-ethereum/index.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ Any participant who broadcasts a transaction request must also offer some amount
 
 The amount of ether paid is a function of the length of the computation. This also prevents malicious participants from intentionally clogging the network by requesting the execution of infinite loops or resource-intensive scripts, as these actors will be continually charged.
 
-## What are dapps? {#what-are-dapps}
+## What are smart contracts? {#what-are-smart-conracts}
```
Comment (wackerow) [inline src/content/developers/docs/intro-to-ethereum/index.md:46]: Sorry, I know I just approved but noticed the typo in the link name, mind fixing that first?

**Review by wackerow — CHANGES_REQUESTED** [review]

File: src/content/developers/docs/intro-to-ethereum/index.md (hunk 1 above)
Comment (rootulp) [reply src/content/developers/docs/intro-to-ethereum/index.md:46]: yea oof, thanks for calling out the typo!

**Review by wackerow — APPROVED** [review]
👍

**Comment by wackerow** [comment]
@all-contributors please add @rootulp for bug patch

**Pull request merged**